acqa-repo1-aws-tf12-part1.tf

```diff
@@ -653,23 +683,23 @@ resource "aws_ecr_repository" "acqa-test-ecr1" {
   image_tag_mutability = "IMMUTABLE"
 
   image_scanning_configuration {
-    scan_on_push = false
+    scan_on_push = true
   }
   tags = {
-    Name = format("%s-ecr1", var.acqaPrefix)
+    Name         = format("%s-ecr1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "ACQA"
+    Owner        = "ACQA"
   }
 }
 
 #ECS Cluster
 resource "aws_ecs_cluster" "acqa-test-ecs1" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure multi-az is configured for AWS ElastiCache clusters**
 
**Vulnerability**
AWS ElastiCache clusters do not have multi-az configured.
 
**Remediation**
Ensure Amazon ElastiCache Redis clusters have the Multi-AZ feature enabled.

source: accurics-